src/model/CollieConfig.ts

```diff
@@ -0,0 +1,49 @@
+import { Logger } from "../cli/Logger.ts";
+
+export class CollieConfig {
+  static CONFIG_FILE_PATH = ".collie/config.json";
+
+  static getFoundation(logger: Logger) {
```

Comment:
d: if we plan on adding more configuration options in the future (Config object), maybe we should already build the interface to return the config object instead, this would make adding more options trivial